openmls/src/group/core_group/proposals.rs

```diff
@@ -430,6 +464,8 @@ impl ProposalQueue {
         for queued_proposal in queued_proposal_list {
             match queued_proposal.proposal {
                 Proposal::Add(_) => {
+                    // `insert` returns true only if there was not entry present
+                    // with the same hash, i.e. if the entry is unique.
```

Comment:
That comment seems unnecessary because the return value is not used.

Reply:
Ah, sorry. That's a leftover.